app/controllers/catalog_controller.rb

```diff
@@ -53,6 +53,11 @@ class CatalogController < ApplicationController
     config.add_show_field "tag_ssim", label: "Tags", link_to_facet: true
     config.add_show_field SolrDocument::FIELD_WORKFLOW_ERRORS, label: "Error", helper_method: :value_for_wf_error
 
+    # exploded_project_tag_ssim indexes all project tag prefixes whereas project tag_ssim only indexes whole tags.
+    # we want to facet on exploded_project_tag_ssim to get the hierarchy.
+    config.add_facet_field "exploded_project_tag_ssim", label: "Project Tag", limit: 9999,
+      component: LazyProjectTagFacetComponent,
+      unless: ->(controller, _config, _response) { controller.params[:no_tags] }
     # exploded_tag_ssim indexes all tag prefixes (see IdentityMetadataDS#to_solr for a more exact
```

Comment:
not exactly this PR's domain, but we should prob eventually get rid of fedora era reference to `IdentityMetadataDS#to_solr`

Reply:
I took care of this with the second commit (use the new exploded_nonproject_tags_ssim field)